src/test/scala/cromwell/util/google/GcsPathSpec.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.util.google
+
+import org.scalatest.{FlatSpec, Matchers}
+
+/**
+ * Spec for GcsPath, in particular 'gs://.../...' URI parsing in constructors.
+ */
+class GcsPathSpec extends FlatSpec with Matchers {
+  final val BUCKET_NAME = "MyBucket"
+  final val SIMPLE_OBJECT_NAME = "object.file"
+  final val NESTED_OBJECT_NAME = "dir1/dir2/object.file"
```

Comment:
Scala constants should be like `BucketName`.

`final` in Scala only makes this field not overridable in a subclass, which I don't think is an issue here.